src/main/java/app/dbService/DBService.java

```
@@ -0,0 +1,66 @@
+package app.dbService;
+
+import app.dbService.dao.BookDAO;
+import app.dbService.dataSets.Book;
+import org.h2.jdbcx.JdbcDataSource;
+
+import javax.annotation.Resource;
+import java.sql.Connection;
+import java.sql.Driver;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+
+public class DBService {
+    private final Connection connection;
+
+    public DBService() {
+        this.connection = getPostgresConnection();
+    }
+
+    public static Connection getPostgresConnection() {
+        try {
+            DriverManager.registerDriver((Driver) Class.forName("org.postgresql.Driver").newInstance());
+            String url = "jdbc:postgresql://localhost:5432/books";
+            String name = "postgres";
+            String pass = "5325475lol";
+
+            JdbcDataSource ds = new JdbcDataSource();
+            ds.setURL(url);
+            ds.setUser(name);
+            ds.setPassword(pass);
+
+
+            Connection connection = DriverManager.getConnection(url, name, pass);
+            return connection;
+        } catch (SQLException | InstantiationException | IllegalAccessException | ClassNotFoundException e) {
+            e.printStackTrace();
+        }
+        return null;
```

In case of error in connection instantiation, you just print an exception and allow for NPE to occur infinitely in DBService#addBook. I would suggest either fail fast ( do not just printStackTrace, but rethrow RuntimeException, that will put your server down) or safely requery Connection (add check for connection being null and reinit it)